src/utils/loaders/google_map_loader.js

```diff
@@ -63,7 +63,7 @@ export default function googleMapLoader(bootstrapURLKeys) {
     );
 
     $script_(
-      `https://maps.googleapis.com/maps/api/js?callback=_$_google_map_initialize_$_${queryString}`,
+      `https://maps.googleapis.com/maps/api/js?callback=_$_google_map_initialize_$_${queryString}&libraries=visualization`,
```

Comment:
Could you add information of why this change?

Reply:
`visualization` library includes the `HeatmapLayer`
Here is the reference https://developers.google.com/maps/documentation/javascript/visualization

Reply:
@ZAKdev That is good, but shouldn't we add this optional if the user sets heatmap as true? Sorry if I am getting it wrong.

Reply:
@itsmichaeldiego, I can check heatmap prop if its not defined then no need to load `visualization` library, but this approach will work only if user has single map on a page because google map api load once for all maps so user may can face some problem.

Reply:
@ZAKdev That is a good catch. I am wondering if adding visualization that includes the `HeatmapLayer` if we don't want Heatmap would make the performance worse 🤔

Reply:
@itsmichaeldiego, not sure about the performance but you are right we should not load this library if user is not using it. Give me some time i will write a solution for it.

Reply:
@ZAKdev That is okey, thanks for your collaboration and your willing to handle this issue. I know it is not a super-easy change but I believe it is totally worth it.

Reply:
@itsmichaeldiego, sorry for the delay, i already tried to resolve this issue.

now i am adding libraries if user providing libraries prop so in future user can add any kind of libraries but now there is another problem. In the beginning google map is already initialized now if i reinitialize google map with libraries so it giving me following error in console.

`You have included the Google Maps API multiple times on this page. This may cause unexpected errors.`

But its working as expected but i know we don't want this error in console so please suggest me or discuss with repo developer then let me know how i should proceed with it.

Reply:
@ZAKdev Let me take a look, but what about if the user just set the library when using `bootstrapURLKeys`? This is called only once when the map is loaded as it contains the key for it, there you can set for example `heatmap: true` and it won't be initialized twice.

Reply:
@itsmichaeldiego, this is what i was doing before, but i am assuming a scenario where user has more then one map and any of the map has heatmap at this point it will throw exception because we are initializing map based on user first map.

If first map is the heatmap then every map will work smoothly.

Reply:
@ZAKdev Oh right, so unless we find a workaround we could request the users to set `heatmap: true` to every map if they want at least one of the maps to have this functionality. But should we written in the docs for sure. 

Otherwise we would have to force the user to share the same `bootstrapURLKeys` in all the maps, which is a really cool thing.

Reply:
@itsmichaeldiego i will commit soon for this.

Reply:
@ZAKdev You rock!